library/rendering/entity_rendering/src/testmod/java/org/quiltmc/qsl/rendering/entity/test/client/ClientEntityRenderingTestmod.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2022 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.rendering.entity.test.client;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import net.minecraft.entity.EquipmentSlot;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.Identifier;
+
+import org.quiltmc.loader.api.ModContainer;
+import org.quiltmc.loader.api.minecraft.ClientOnly;
+import org.quiltmc.qsl.base.api.entrypoint.client.ClientModInitializer;
+import org.quiltmc.qsl.rendering.entity.api.client.ArmorRenderingRegistry;
+import org.quiltmc.qsl.rendering.entity.test.EntityRenderingTestmod;
+
+@ClientOnly
+public final class ClientEntityRenderingTestmod implements ClientModInitializer,
+		ArmorRenderingRegistry.TextureProvider {
+	@Override
+	public void onInitializeClient(ModContainer mod) {
+		// TEMP DISABLED TO TEST getTexture() METHOD
```

Comment:
Hm, while this does say that one will be enabled over the another, I do wonder if we can somehow use both on the armor for testing purposes

Reply:
OH RIGHT forgot about this
I'll make the leggings texture have a nonstandard name and use the TextureProvider for that :P